### PR TITLE
Bugfix/pop template lint errors

### PR DESCRIPTION
**Pull request by ricominten**
Fixed all but 1 linting error when creating new components and React adapters using yarn generate.

### CODE REVIEW

File: plopTemplates/reactAdapters/index.stories.tsx.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import {{pascalCase name}} from './lib';
+import {storiesOf} from '@storybook/react';
+import {{{pascalCase name}}} from './lib';
```
Comment (ricominten) [inline plopTemplates/reactAdapters/index.stories.tsx.hbs]: Still haven't figured out how I can make this line work

Comment (ricominten) [inline plopTemplates/reactAdapters/index.stories.tsx.hbs]: I still haven't figured out how to fix this one

Comment (jennasalau) [reply plopTemplates/reactAdapters/index.stories.tsx.hbs]: https://handlebarsjs.com/expressions.html#whitespace-control

So should be this i think

```
{ {{~pascalCase name~}} }
```

Comment (ricominten) [reply plopTemplates/reactAdapters/index.stories.tsx.hbs]: Thanks! included this in the latest commit

**Pull request merged**